.github/workflows/images.yml

```diff
@@ -0,0 +1,30 @@
+name: Container Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        description: Name of the image to build and push to quay.io 
+        required: true
+        type: choice
+        options:
+        - build_base
+        - integration_test_base
+
+jobs:
+  build_container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Quay login
+        run: |
+          echo ${{ secrets.QUAY_BOT_API_TOKEN }} | buildah login -u="bluechi+bluechi_bot" --password-stdin quay.io
```

Comment:
Wouldn't the API token be visible in the execution logs of the workflow?

Reply:
Maybe a better way to implement this is to split the build_and_push script so that it only builds the image and then use https://github.com/marketplace/actions/push-to-registry to push it

Reply:
> Wouldn't the API token be visible in the execution logs of the workflow?
@mwperina Yes and no. Its GitHub Action specific, but they will mask any secrets. 

@ygalblum Good point. I was too lazy to refactor the script, to be honest. But lets use that action and not rely solely on gh masking secrets.

Reply:
@mwperina @ygalblum Could you have another look?
I know there is also https://github.com/marketplace/actions/buildah-build for building the images/manifest, but wanted to still use the bash script.